src/aragonjs-wrapper.js

```diff
@@ -58,6 +61,7 @@ const applyAppOverrides = apps =>
 // Sort apps, apply URL overrides, and attach data useful to the frontend
 const prepareFrontendApps = (apps, gateway) => {
   return applyAppOverrides(apps)
+    .filter(app => app.appId !== EVMSCRIPT_REGISTRY_APP_ID)
```

Comment:
I think we can keep showing this app, but note it as a core app.

Reply:
I hardcoded it for now:

https://github.com/aragon/aragon/blob/03a6002661abd057baf637f2942ab77b6d385b19/src/aragonos-utils.js#L3-L12
https://github.com/aragon/aragon/blob/03a6002661abd057baf637f2942ab77b6d385b19/src/aragonjs-wrapper.js#L73

Reply:
:+1: we can also add a hardcoded role for it too:

https://github.com/aragon/aragonOS/blob/v3.1.12/contracts/evmscript/EVMScriptRegistry.sol#L17

(Although this will need to be changed with aragonOS 4, but hopefully aragon.js will just handle this by then).